coordinator/clientapi/clientapi.go

```diff
@@ -741,3 +758,41 @@ func (a *ClientAPI) WriteSecrets(ctx context.Context, rawSecretManifest []byte,
 
 	return commit(ctx)
 }
+
+// SignQuote verifies the quote and signs it with the Coordinator's root key.
+func (a *ClientAPI) SignQuote(ctx context.Context, quote []byte) (signedQuote []byte, tcbStatus string, err error) {
+	a.log.Info("SignQuote called")
+	defer func() {
+		if err != nil {
+			a.log.Error("SignQuote failed", zap.Error(err))
+		} else {
+			a.log.Info("SignQuote successful")
+		}
+	}()
+
+	// Verify the quote
+	var verifyErr error
+	report, err := enclave.VerifyRemoteReport(quote)
```

Comment:
This API should accept both OE reports and raw SGX quotes. For the difference, see
https://github.com/edgelesssys/marblerun/blob/3e62cf3e2976ef09cdb3944edd719b61ac733f1b/marble/premain/util.go#L11-L17